doc/src/manual/noteworthy-differences.md

```diff
@@ -158,6 +158,13 @@ For users coming to Julia from R, these are some noteworthy differences:
   * In Julia, values are not copied when assigned or passed to a function. If a function modifies an array, the changes
     will be visible in the caller. This is very different from R and allows new functions to operate
     on large data structures much more efficiently.
+  * In Julia, a function argument is either positional, or a keyword
+    argument. Unlike in R -- where any argument can be passed either
+    positionally or by name -- in Julia, positional arguments cannot be
+    passed by name, and keyword arguments cannot be passed positionally.
+    E.g. a function with signature `work(x; T)` must be called
+    as `work(4, T = 5)`, and not `work(4, 5)` or `work(x = 4, T = 5)`.
```

Comment:
nit: probably should recommend `work(4; T = 5)` (note the semicolon there). both formulations function the same, but may as well set the cleanest example